package/scripts/aca/certificate_generate.sh

```diff
@@ -75,6 +75,10 @@ if [ $DOCKER_CONTAINER = true ]; then
     if [[ $(ss -t -l -n | grep -q LISTEN.*:::8009) -eq 0 ]]; then
         echo "Tomcat is running, so we restart it."
         /usr/libexec/tomcat/server stop
+        # Wait for Tomcat to stop completely and prevent port bind collisions
+        while [ -z $(tail -n 1 /var/log/tomcat/catalina.*.log | grep "Destroying ProtocolHandler \[\"http-bio-8443\"\]") ]; do
```

Comment:
There are two things to be concerned about here:
1. Because it checks all log files in the directory, the `while` condition is going to tell us if _any_ log file exists from a time when tomcat was stopped. Isn't that too broad? We want to know if it is stopped now -- not ever.
1. The `$()` needs double quotes around it or there will be an error message about "too many arguments". (`-z` only takes a single argument.)

Reply:
1) This is only ever going to run when the ACA is deployed in a Docker container. Presumably each new instance of the Docker container will install the ACA on startup, so it's unlikely there should ever be anymore than the current day's worth of catalina.*.log. That being said, if you know of a good way to get today's date in the catalina format we can update that.

2) It doesn't appear to need double quotes, and it seems to be running just fine. If it didn't then my Docker containers would get trapped in this while loop or at the very least throw the error you suggest.

Reply:
I just pushed an update that should resolve your first comment.